actions/image_partition_action.go

```diff
@@ -668,6 +668,11 @@ func (i *ImagePartitionAction) Verify(context *debos.DebosContext) error {
 
 		for pidx, _ := range i.Partitions {
 			p := &i.Partitions[pidx]
+
+			if strings.ToLower(p.FS) == "none" {
```

Comment:
you didnt test this did you?

consider this action for a test:
```
 - action: image-partition
   imagename: "test.img"
   imagesize: 1GB
   partitiontype: gpt
   mountpoints:
     - mountpoint: /
       partition: root
     - mountpoint: /boot
       partition: boot
   partitions:
     - name: boot
       fs: fat32
       start: 0%
       end: 64MB
     - name: test
       fs: none
       start: 64MB
       end: 128MB
     - name: root
       fs: ext4
       start: 128MB
       end: 100%
```

from a quick look it will fail for every partition which has the `none` type - even those which aren't being mounted. so you need to check the partition type AFTER you have found the correct partition for this mountpoint.

so i think the partition type check should probably go after `if m.part == nil {`

Reply:
> you didnt test this did you?

I did test this with yaml file in https://github.com/eballetbo/debos-recipes/blob/master/images/weston/debian-qemu-uefi.yaml and it exited with error when fs was none.

Let me check with the one you provided.

Reply:
Updated the error message to print partition name also. I checked with the above file and it exited if fs is none for each partition.

Reply:
Right, the recipe i posted about should not fail because `test` isn't ever mounted :-)

hence why we need to do the check for mountpoints and not partitions.

Reply:
Sorry, I missed the point before :)
Will make the changes and test it.